src/v0/sources/cordial/transform.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const fs = require('fs');
+const md5 = require('md5');
+const Message = require('../message');
+const { CommonUtils } = require('../../../util/common');
+const { generateUUID, isDefinedAndNotNull } = require('../../util');
+const { eventsMapping } = require('./config');
+
+const mapping = JSON.parse(fs.readFileSync(path.resolve(__dirname, './mapping.json'), 'utf-8'));
```

Comment:
you can use `require` for this.

`const mapping = require('./mapping.json');`

Reply:
@koladilip done.